src/presenters/business/business-email-change-presenter.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Formats data ready for presenting in the `/business-details` page
```

Comment:
The page this presenting data for is the `/business-email-change`

```suggestion
 * Formats data ready for presenting in the `/business-email-change` page
```

Reply:
comments removed